about.markdown

```diff
@@ -1,13 +1,31 @@
 ---
-layout: fullwidth
+layout: fullwidth-with-breadcrumbs
 primary_title: About OpenSearch
-title: About
----
-
-_Updated December 20, 2022_
-
-[Principles for development](#principles-for-development) &middot; [Founding Documents](#founding-documents) &middot; [OpenSearch disambiguation](#opensearch-disambiguation) &middot;
```

Comment:
@astephanus - per our conversation last week, please make sure these anchor links are back in the page as well as the Updated date above. Thanks

Reply:
@krisfreedain I have worked with Maeve on a design for that and have updated the PR accordingly.